apps/sentry/src/common/env.ts

```diff
@@ -0,0 +1,25 @@
+import { z } from 'zod';
+
+const zEnvInt = () => z.coerce.number().int().positive();
+
+export const env = z
+  .object({
+    ELBA_API_KEY: z.string().min(1),
+    ELBA_API_BASE_URL: z.string().url(),
+    ELBA_REDIRECT_URL: z.string().url(),
+    ELBA_SOURCE_ID: z.string().uuid(),
+    ELBA_WEBHOOK_SECRET: z.string().min(1),
+    ENCRYPTION_KEY: z.string().min(1),
+    DATABASE_URL: z.string().min(1),
+    DATABASE_PROXY_PORT: zEnvInt().optional(),
+    VERCEL_ENV: z.string().min(1).optional(),
+    SENTRY_APP_INSTALL_URL: z.string().url().default('https://sentry.io/oauth'),
```

Comment:
```suggestion
    SENTRY_APP_INSTALL_URL: z.string().url(),
```

Let's not set a default value here, it's not really useful.